lighthouse-core/gather/gatherers/inspector-issues.js

```diff
@@ -33,7 +33,9 @@ class InspectorIssues extends Gatherer {
   async beforePass(passContext) {
     const driver = passContext.driver;
     driver.on('Audits.issueAdded', this._onIssueAdded);
-    await driver.sendCommand('Audits.enable');
+    try {
+      await driver.sendCommand('Audits.enable');
+    } catch (_) {} // Fails if Chrome is old.
```

Comment:
can you change this to an "Fails if Chrome is older than mXX" for easy future reference? (could also be a TODO to remove but that may be overkill)

Reply:
ref https://chromiumdash.appspot.com/commits?commit=f51add1&platform=Windows

Reply:
it should be good >= 82 .... gonna comment out and see if it still fails in CI (it shouldnt..)

Reply:
appveyor fails bc it's holding on to an old version of chrome. no idea how to clear cache so I'll just push a `rm `...

Reply:
lol I can't even figure out how to get the version of chrome from appveyor.


![image](https://user-images.githubusercontent.com/4071474/85634223-bedb9080-b62f-11ea-9b00-f300631a69a3.png)

![image](https://user-images.githubusercontent.com/4071474/85634211-ba16dc80-b62f-11ea-9a60-4442def6aad2.png)

Reply:
blocked on #11017

Reply:
> lol I can't even figure out how to get the version of chrome from appveyor

Not a reason to keep our appveyor on life support, but FWIW this is because Chrome on Windows isn't a console app. See https://crbug.com/158372 for some awkward ways to get a version number from the command line there.

Reply:
> "Fails if Chrome is older than mXX"

not a huge deal but my brain associates "old"/age  with higher numbers, but here, older means lower. 😖 

maybe we just stick to `if Chrome is < m82` or w/e